ui/component/app/view.jsx

```diff
@@ -284,80 +275,41 @@ function App(props: Props) {
       ref={appRef}
       onContextMenu={IS_WEB ? undefined : e => openContextMenu(e)}
     >
-      <Router />
-      <ModalRouter />
-      <FloatingViewer pageUri={uri} />
-
-      {/* @if TARGET='web' */}
-      <YoutubeWelcome />
-      <OpenInAppLink uri={uri} />
-      {/* @endif */}
-
-      {/* @if TARGET='app' */}
-      {showUpgradeButton && (
-        <Nag message={__('An upgrade is available.')} actionText={__('Install Now')} onClick={requestDownloadUpgrade} />
-      )}
-      {/* @endif */}
-      {/* @if TARGET='web' */}
-      {currentlyDegradedPerformance && (
-        <Nag
-          type="error"
-          message={
-            <I18nMessage
-              tokens={{
-                more_information: <Button button="link" label={__('more')} href="https://status.lbry.com/" />,
-              }}
-            >
-              lbry.tv performance may be degraded. You can try to use it, or wait 5 minutes and refresh. Please no crush
-              us.
-            </I18nMessage>
-          }
-          actionText={__('Refresh')}
-          onClick={() => window.location.reload()}
-          onClose={() => setCurrentlyDegradedPerformance(false)}
+      {IS_WEB && lbryTvApiStatus === STATUS_DOWN ? (
+        <Yrbl
+          className="main--empty"
+          title={__('lbry.tv is currently down')}
+          subtitle={__('Something about fixing the squeeky wheel or something')}
```

Comment:
@kauffj I forget what you said this label should be.

Reply:
if I had something before I forget

how about:

"My wheel broke, but the good news is that someone from LBRY is working on it."